pulpcore/pulpcore/app/models/publication.py

```diff
@@ -20,6 +42,68 @@ class Publication(Model):
 
     repository_version = models.ForeignKey('RepositoryVersion', on_delete=models.CASCADE)
 
+    complete = False  # placeholder
```

Comment:
dont forget to make this a model.Boolean

Reply:
The complete field will be added by #3299.  This just a placeholder to get it working that will be removed before merge.